lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -51,11 +51,12 @@ def check_time_node(klass, node)
                     !(chain & good_methods).empty?
 
           method_name = (chain & DANGER_METHODS).join('.')
+          safe_method_name = safe_method(method_name, node)
 
           add_offense(node, :selector,
                       format(MSG,
                              "#{klass}.#{method_name}",
-                             "#Time.zone.#{method_name}")
+                             "#Time.zone.#{safe_method_name}")
```

Comment:
I fix it. Now properly colored in the terminal.

![screen shot 2015-04-10 at 12 33 24 am](https://cloud.githubusercontent.com/assets/9650/7070501/5ce21fa8-df1a-11e4-883c-4124b4193a8f.png)